autorest/adal/msi_windows.go

```diff
@@ -0,0 +1,6 @@
+// +build windows
+
+package adal
+
+// msiPath is the path to the MSI Extension settings file (to discover the endpoint)
+var msiPath = "C:/WindowsAzure/Config/ManagedIdentity-Settings"
```

Comment:
We might want to use environment var SystemDrive instead of C:

Reply:
Fixed